src/containers/stage-selector.jsx

```diff
@@ -130,8 +130,8 @@ StageSelector.propTypes = {
     onSelect: PropTypes.func
 };
 
-const mapStateToProps = (state, {assetId, id}) => ({
-    url: assetId && state.scratchGui.vm.runtime.storage.get(assetId).encodeDataURI(),
```

Comment:
Other side of the change — we don't have to pull anything out of storage anymore, it can just come directly from the costume/sound on the VM.